pxblue/react-design-patterns
# Feature/pxblue 2543 action list inline

Fixes # PXBLUE-2543


#### Changes proposed in this Pull Request:
- Added list with inline actions
- Added mobile view styles
- Updated UI test cases


#### Screenshots:
**Desktop**
<img width="1657" alt="Screenshot 2021-11-12 at 5 42 55 PM" src="https://user-images.githubusercontent.com/25982779/141465630-738120c9-446e-40b1-aab4-06cedb07d068.png">

**Mobile**
<img width="318" alt="Screenshot 2021-11-12 at 5 47 43 PM" src="https://user-images.githubusercontent.com/25982779/141466283-79b2556b-fd7a-4813-ae83-32dee728c2c7.png">


#### To Test:
- http://localhost:3000/inline-actions
- Hover on any of the items in Desktop view to see actions
- Click on any of the item's morevert icon in Mobile view to see actions
